microk8s-resources/actions/common/utils.sh

```diff
@@ -104,6 +104,16 @@ get_default_ip() {
     fi
 }
 
+get_ips() {
+    local IP_ADDR="$($SNAP/bin/hostname -I)"
```

Comment:
I think `hostname` is not under  `$SNAP/bin/hostname`. Please add it under https://github.com/ubuntu/microk8s/blob/master/snapcraft.yaml#L132